### PR TITLE
make hw4

### PR DESCRIPTION
**Pull request by Kiril99**
hw4 -> done

### CODE REVIEW

File: HW4/src/isPrime.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+function isPrime(x) {
+    if (x/x === 1 && x/1 === x){
```
Comment (vakal237078) [inline HW4/src/isPrime.js]: 1 is not a prime !
But your function when take 1 return true

File: HW4/src/reverseNumber.js [hunk 2]
```diff
@@ -0,0 +1,5 @@
+function reverseNumber(x) {
+    var result = x.toString().split("").reverse().join("")
```
Comment (vakal237078) [inline HW4/src/reverseNumber.js]: when your function accepts a negative number it returns a positive number

File: HW4/src/reverseNumber.js [hunk 3]
```diff
@@ -0,0 +1,5 @@
...
+    return Number(result);
+}
```
Comment (Davydovychyurii) [inline HW4/src/reverseNumber.js:3]: not a valid result with a negative value

File: HW4/src/getMin.js [hunk 4]
```diff
@@ -0,0 +1,10 @@
+function getMin(a,b,c,d){
+    var result;
+    for (var i =0; i< arguments.length; i++){
+        if (arguments[i]<arguments[i+1] || arguments[i]<arguments[i-1]){
+            result =arguments[i];
+        }
+    }
```
Comment (Davydovychyurii) [inline HW4/src/getMin.js:7]: nothing is displayed in the console

File: HW4/src/getClosetstToZero.js [hunk 5]
```diff
@@ -0,0 +1,10 @@
+function getClosestToZero(a,b,c,d,e) {
+    var goal = 0;
+    var counts  = Array.prototype.slice.call(arguments);
+    var closest = counts.reduce(function(prev, curr) {
+        return (Math.abs(curr - goal) < Math.abs(prev - goal) ? curr : prev);
```
Comment (Davydovychyurii) [inline HW4/src/getClosetstToZero.js:5]: nothing is displayed in the console

File: HW4/src/reverseNumber.js (hunk 3 above)
Comment (Kiril99) [reply HW4/src/reverseNumber.js:3]: +

File: HW4/src/getMin.js (hunk 4 above)
Comment (Kiril99) [reply HW4/src/getMin.js:7]: +

File: HW4/src/getClosetstToZero.js (hunk 5 above)
Comment (Kiril99) [reply HW4/src/getClosetstToZero.js:5]: +

**Review by Davydovychyurii — APPROVED** [review]

File: HW4/src/isPrime.js (hunk 1 above)
Comment (Kiril99) [reply HW4/src/isPrime.js]: +

File: HW4/src/reverseNumber.js (hunk 2 above)
Comment (Kiril99) [reply HW4/src/reverseNumber.js]: +

**Review by vakal237078 — APPROVED** [review]

**Pull request merged**